src/app/layouts/header/export-popup/export-popup.component.ts

```diff
@@ -54,6 +66,24 @@ export class ExportPopupComponent implements OnInit, OnDestroy {
         this.popup.hide();
     }
 
+    displayfileContent(currentTabIndex) {
+        if(currentTabIndex === 0) {
+            this.buttonDisabled = this.settingDisabled = this.loadIndicatorVisible = true;
+            this.importService.exportCss(this.schemeName, this.makeSwatch).then(css => {
```

Comment:
I think we should improve previously exportMeta and exportCss methods - if Apply button is disabled, we can get css/meta from textarea without the help of importService. After exporting to the file we can fill the textarea with the data.